back/spec/services/xlsx_service_spec.rb

```diff
@@ -109,7 +109,9 @@ def xlsx_to_array(xlsx, sheet_index: 0)
       let(:xlsx) { service.generate_ideas_xlsx(ideas, view_private_attributes: false) }
 
       it 'hides private attributes' do
+        custom_field = create :custom_field
         expect(worksheet[0].cells.map(&:value)).not_to include 'author_email'
```

Comment:
Wonder if it would be worth just beefing up these tests a little to check that it doesn't include any of the other personal data such as first_name and last_name - is it just those fields and email?